src/common/config.cpp

```diff
@@ -552,10 +552,22 @@ void load(const std::filesystem::path& path) {
         fmt::print("Got exception trying to load config file. Exception: {}\n", ex.what());
         return;
     }
+
     if (data.contains("General")) {
+
+        toml::value& table = toml::find(data, "General");
+        auto findkey = table.as_table().find("isPS4Pro");
+        if (findkey != table.as_table().end()) {
+            table.as_table().erase(findkey);
+            data["General"]["isPS4ProEnabled"] = false;
+            std::ofstream file(path, std::ios::binary);
+            file << data;
+            file.close();
+        }
```

Comment:
No need to rename the option in my opinion, you can remove this and go back the previous name.

Reply:
The intent behind the rename is to "reset" the setting for current users that have this on, since if we remove it from the GUI without doing this, it would always be on for the users that currently have it on. The first version of this PR had pro mode hardcoded to false to prevent this, but I changed it to what it is now so that users can still enable pro mode by manually editing the config file

Reply:
Gotcha, I understand thats not something you'd not want in there. Okay, will revert the changes except for removing the setting in the GUI and deleting the duplicate line in config::save :)